### DISCUSSION TITLE
Just discovered DAX Studio

### DISCUSSION THREAD

**Discussion by learnpython2017** [DaxStudio/DaxStudio#736] [General]
cannot connect SQL server on my laptop to DAX, see image for error.  Thank you
![DAXconnectIssue](https://user-images.githubusercontent.com/29578556/137028076-78ed4d6d-6f50-4662-b51c-ee949a43feb1.jpg)
![image](https://user-images.githubusercontent.com/29578556/137028105-eca031f1-040e-46ae-950a-b168a7fab9a1.png)

**Comment by dgosbell** [comment]
That would be correct. DAX Studio cannot connect to SQL Server as it does not support the DAX language. We can connect to Analysis Services, Azure Analysis Services, Power BI Desktop, Power BI XMLA endpoint and Excel Power Pivot models.

**Reply by learnpython2017** [reply]
thank you very much!
